### DISCUSSION TITLE
Centering Twitter Widget on Worpress Page

### DISCUSSION THREAD

**Discussion by VLS77589** [WordPress/gutenberg#48601] [Q&A]
Hello, and sorry if this isn't the right place to post. I'm building a Wordpress page and inserted a Twitter Widget and selected the "centered" option but it shows up as off-center both in preview and once the site is saved (too far left in comparison to other centered text boxes above it). 

Here is the code for it (included spaces to be able to show it, on the site they are not present), clearly I know very little about how to actually code any pointers or edits are helpful. It's confusing since the code itself indicates "aligncenter", but another adjustment could be changing the width, however I'm not certain of how modify the existing code for that either.

< figure class="wp-block-embed aligncenter is-type-rich is-provider-twitter wp-block-embed-twitter">< div class="wp-block-embed__wrapper" >
 https://twitter.com/rensselaervc
< /div >< /figure >

Here's a screenshot of what I mean, see comparison to above white text/content box, that is wider and actually in center position.

<img width="1016" alt="Screenshot 2023-02-28 at 8 51 26 AM" src="https://user-images.githubusercontent.com/126597762/221921913-20e11bc6-f092-4d06-8603-fc1b6a338053.png">

**Comment by ghost** [comment]
<img width="281" alt="Screenshot 2023-03-05 at 11 04 09 PM" src="https://user-images.githubusercontent.com/118618474/222988503-fd87bc16-212f-470e-b055-150c091dda99.png">

@VLS77589 It would be better to open an issue for this! :)
